dev/com.ibm.ws.app.manager.springboot/src/com/ibm/ws/app/manager/springboot/util/SpringBootThinUtil.java

```diff
@@ -811,7 +824,13 @@ public static Set<String> getStarterArtifactIds(String starter) {
                                                                                            "org.apache.tomcat.embed:tomcat-embed-core:jar:10.1.42:compile",
                                                                                            "org.apache.tomcat.embed:tomcat-embed-el:jar:10.1.42:compile",
                                                                                            "org.apache.tomcat.embed:tomcat-embed-websocket:jar:10.1.42:compile");
-
+        private final static List<String> mvnSpringBoot40TomcatStarterDeps = Arrays.asList(
+                                                                                           "org.springframework.boot:spring-boot-starter-tomcat:jar:4.0.0-M1:compile",
```

Comment:
Do we know how this code will behave once they have a released version of the artifacts vs. the mile stone?  Do we know if this will continue to work then?  What about if there is an `M2`?

Reply:
The code looks for only artifact id when filtering the starter jars.